src/test/model/graph.test.ts

```diff
@@ -228,7 +228,7 @@ describe('Graph', () => {
             assert.strictEqual(node?.props[1].key, 'b');
         });
 
-        it.skip('removes second link of two looped nodes', async () => {
+        it('removes one of the links of two looped nodes', async () => {
```

Comment:
'removes the first link of two looped nodes'

I prefer, since it would have to always be stably so in order for the test to pass.
If better not - then why not?